mintapi/tests.py

```diff
@@ -50,7 +50,9 @@ def tearDown(self):  # noqa
         mintapi.api.Mint = self._Mint
 
     def test_accounts(self):
-        accounts = mintapi.get_accounts('foo', 'bar', ius_session='baz')
+        accounts = mintapi.get_accounts('foo', 'bar', session_cookies={
```

Comment:
@jprouty it looks like you are testing a call to `get_accounts` with a different method signature, passing in the new `session_cookies` parameter. Shouldn't this also be updated in the main `api.py` file for handling the `--accounts` switch? I think it is here: https://github.com/jprouty/mintapi/blob/ce19db9b83c38d9d899256eef57c9db2cd7fe542/mintapi/api.py#L728

Reply:
Based on a quick read, I'm pretty sure this flow is unchanged from before. The code you're commenting on is the module function, not the class method on Mint. The main script first constructs a Mint object (with session cookies if given, or gets them from Mint) then does a get_accounts on that client instance.